src/components/NavigationBar/PNavigationBar.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="p-navigation-bar" :class="classes.root">
+    <template v-if="$slots.prepend">
```

Comment:
I'd prefer if we used `useSlots` rather than the `$slots` var cause its kinda magical.

Reply:
oh yeah true, didn't even realize i did that - updated